src/components/VerticalListMenu/VerticalListMenu.spec.jsx

```diff
@@ -0,0 +1,160 @@
+import _ from 'lodash';
```

Comment:
Hmm... interesting... my pattern for tests has been primarily around the props and the event handlers.